docs/content/recipes/federation.md

```diff
@@ -19,6 +19,16 @@ federation:
   package: generated
 ```
 
+### Federation 2
+
+If you are using Apollo's Federation 2 standard, your schema should automatically be upgraded so long as you include the required `@link` directive within your schema. If you want to force Federation 2 composition, the `federation` configuration supports a `version` flag to override that. For example:
+
+```yml
+federation:
+  filename: graph/generated/federation.go
+  package: generated
```

Comment:
Did you want to add a `version: 2` key here?

Reply:
We can implicitly pull that from the schema, hence why I left it out; might be worthwhile, however.